NewForce-Cohort-6/tabloidfullstack-mashed-potatoes
# 20CreateComment

# Description
[Trello Ticket #20 (https://trello.com/c/TttXDlx9)]

Added feature to add a new comment to a post. The comments can be added by clicking the add comment link at the bottom of the post details page. There's also a link to return to the post from the comments, a link to view comments and a link to return to the post list from the post details page.

# Testing Instructions
1. `git add` and `git commit` whatever you're working on on your branch
2. `git checkout main`
3. `git fetch --a`
4. `git checkout 20CreateComment`
5. If you don't see the most recent version of my files, `git pull origin 19ViewPostComments'
6. If you haven't created the Tabloid database, please do so.
Use this data to fill out tables and columns: [SQL Tables](https://github.com/NewForce-Cohort-6/tabloidfullstack-mashed-potatoes/blob/main/SQL/01_Tabloid_Create_DB.sql)
Use this to seed the database : [SQL SEED DATA](https://github.com/NewForce-Cohort-6/tabloidfullstack-mashed-potatoes/blob/main/SQL/02_Tabloid_Seed_Data.sql)
7. Click the execute with debugger solid green button to launch the app
8. Log into the website using “leo@gmail.com” if greeted with a login page
9. From the home page, click the “Post or My Post” option in the navbar
10. A list of posts will show on the page.
11. Choose a post to view. This will take you to the post details page with options to view comments, return to the post list or add a comment at the bottom of the page.
12. Click the add comment link. This will take you to a form to enter details about your comment. Fill in the text boxes for subject and content and click the save button. This will take you back to the comments list with the new comment included in the list. You can also cancel making a new comment by clicking the back to post button which will take you back to the list of comments without adding the cancelled comment.
13. After viewing or adding new comments click the return to post link which will return you to post and then you can click the link to return to the post list. 



# Checklist:
